receiver/redisreceiver/proto.go

```diff
@@ -24,11 +24,15 @@ import (
 
 // Helper functions that produce protobuf
 
-func newMetricsData(protoMetrics []*metricspb.Metric) *consumerdata.MetricsData {
+func newMetricsData(protoMetrics []*metricspb.Metric, serverName string) *consumerdata.MetricsData {
+	labels := map[string]string{"type": typeStr}
+	if serverName != "" {
+		labels["server_name"] = serverName
```

Comment:
We should follow the semantic conventions defined here (if this can be used as service name):
https://github.com/open-telemetry/opentelemetry-specification/tree/master/specification/resource/semantic_conventions#service

Reply:
I wonder if it makes sense to define these names somewhere so that developers can browse and use them without the possibility of making a mistake.

Reply:
They are defined here: https://docs.google.com/presentation/d/1Q6CjinzIG5D6BweCpj-_AFQG0V0NmVkAgVPfhj0G8XQ/edit?usp=sharing

Reply:
Thank you for the link Bogdan, I will take a look. What I was wondering was if it might be helpful to capture these conventions in a package so that developers can just say, for example, `conventions.ServiceName` or something rather than `"service.name"`.